categories/javascript.json

```diff
@@ -40,5 +40,16 @@
       "playground",
       "snippets"
     ]
+  },
+  {
+    "name": "The Modern Javascript Tutorial",
```

Comment:
Can you capitalise the ”S” in JavaScript please. :)

Reply:
UpperCamelCase it is. :+1: